### ISSUE TITLE
drive letter casing is inconsistent with node 0.11x

### ISSUE THREAD

**Issue by tkellen** [browserify/resolve#61]
in node 0.11.14, executing

``` js
var resolve = require('resolve');
console.log(require.resolve('resolve'));
console.log(resolve.sync('resolve'));
```

outputs:

```
E:\test\node_modules\resolve\index.js
e:\test\node_modules\resolve\index.js
```

ref gulpjs/gulp#664

**Comment by ljharb** [comment]
https://github.com/gulpjs/gulp/issues/664#issuecomment-69487133 implies that this behavior would require `resolve` to *un*normalize things, since calling into `path.join` is necessary for windows compatibility. Is this just about drive letters (which would be relatively trivial to capitalize), or are there other differences we'd run into trying to fix it piecemeal?

**Comment by ljharb** [comment]
Per https://github.com/nodejs/node-v0.x-archive/pull/7799#issuecomment-132149227, this was fixed in https://github.com/nodejs/node-v0.x-archive/commit/016e08458cc38d7324c907c6594f2b56a5a19719, which means it's fixed in v0.11.15.

Given that v0.11.14 is the sole version this is broken in, and 0.11 is both long since EOL and was unstable in its entire line, I'm going to close this as a wontfix.
